src/components/KeyboardNavigation/Provider.js

```diff
@@ -29,14 +29,16 @@ export default class KeyBoardProvider extends React.Component {
     role: PropTypes.string.isRequired,
     className: PropTypes.string,
     keywordSearch: PropTypes.bool,
-    keyBoardRef: PropTypes.func
+    keyBoardRef: PropTypes.func,
+    ariaLabelledby: PropTypes.string
   };
 
   static defaultProps = {
     children: null,
     keyBoardRef: null,
     className: null,
-    keywordSearch: false
+    keywordSearch: false,
+    ariaLabelledby: null
```

Comment:
Why default value is null since the type is string?

Reply:
Because we don't need it to be set by default, just when comes from props as it is optional

Reply:
I think empty string will do the same job